_episodes/10-data-transformation.md

```diff
@@ -59,7 +59,7 @@ Such tests can be combined with other GREL expressions to create more complex tr
 >1. Make sure you have already split the author names into individual cells using ```Edit cells->Split multi-valued cells``` (you should have done this in exercise 5)
 >2. On the Authors column, use the dropdown menu and select ```Facet->Custom text facet...```
 >3. The Custom text facet function allows you to write GREL functions to create a facet
->4. In the Expression box type ```value.contains(",").toString()```
+>4. In the Expression box type ```value.contains(",")```
```

Comment:
Removing the 'toString()' here will prevent the lesson working with OpenRefine 3.1. Leaving it in makes the instruction work in all versions of OpenRefine. I don't have an issue with removing it but just noting in case you think it's worth leaving in

Reply:
[See above](https://github.com/LibraryCarpentry/lc-open-refine/pull/242#discussion_r962266503)